backend/src/openarchiefbeheer/destruction/models.py

```diff
@@ -271,6 +271,9 @@ def create_report_zaak(self) -> None:
 
         attach_report_to_zaak(destruction_list, store)
 
+    def clear_local_metadata(self) -> None:
+        self.items.filter(status=ListItemStatus.suggested).update(extra_zaak_data={})
```

Comment:
Do we want to filter here, it feels that this might easily be overlooked in the future. And the test does not seem to cover that case.

I would expect that either clear_local_metadata clears local meta data regardless of the status, or that a tests exists that checks if at the end of the flow no such data exists.